dev/github/run_checks.sh

```diff
@@ -27,7 +27,13 @@ export MAVEN_OPTS="-Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simple
 # Always use java 8 to compile the source code
 JAVA_HOME=/usr/local/openjdk-8
 PATH=$JAVA_HOME/bin:$PATH
-mvn -Duser.home=/home/jenkins -T 4C clean install -DskipTests
+
+profiles=""
```

Comment:
`run_checks.sh` is currently only called by the `checkstyle.yml` file. the new `native_unit_tests.yml` calls `run_tests.sh` instead, so this change does not take effect by any existing workflow.

i would prefer to not include this to minimize the clutter in the shell scripts as they have been very messy to navigate and refactor when necessary (note that this is exclusively due to using shell; for any other navigable or compiled code, i would also prefer completeness). a recent point of evidence: i broke the integration tests when trying to recreate these files for 3.x/main branch